ieeeuoft/hackathon-template
# Ieee 262 update the teams model to have a project description field

## Overview

- Resolves #[IEEE-259](259)
- Added an input box on team's home page to describe what their project is


## Unit Tests Created

- Tests were created to verify that the Redux store updates correctly with the new project_description after dispatching the updateProjectDescription thunk.


## Steps to QA

- Navigate to the dashboard page
- There should be a textbox with text saying "Write your project description here"



[IEEE-259]: https://ieeeuoft.atlassian.net/browse/IEEE-259?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
